spec/models/concerns/digital_object_concerns/attribute_assignment/doi_spec.rb

```diff
@@ -71,4 +72,31 @@
       end
     end
   end
+  describe "#ensure_doi_if_requested" do
+    let(:mint_doi) { true }
+    before { digital_object_with_doi.instance_variable_set('@mint_doi', mint_doi) }
```

Comment:
```suggestion
    before { digital_object_with_doi.mint_doi = mint_doi }
```
Since this is the way we'd generally set that instance variable in the app, I figured it would be good to do that for this test too.